ephios/core/templatetags/event_extras.py

```diff
@@ -77,3 +80,14 @@ def event_signup_state_counts(event, user):
                 counter[participation.state] += 1
                 break
     return counter
+
+
+@register.simple_tag(name="event_bulk_actions")
+def event_bulk_actions():
+    return format_html(
+        reduce(
+            lambda result, item: result + "".join(item[1]),
```

Comment:
Can you use nested tuple unpacking? 
```suggestion
            lambda html, (receiver, buttons): html + "".join(buttons),
```

Reply:
tuple parameter unpacking is not supported in python3

Reply:
Oh okay. Do you see another way to make that more readable?

Reply:
is this comment based on the new code?

Reply:
Probably not